downstream/modules/platform/proc-add-operator-execution-nodes.adoc

```diff
@@ -0,0 +1,48 @@
+[id="add-operator-execution-nodes_{context}"]
+
+.Prerequisites
+* An {ControllerName} instance
+* The receptor collection package is installed
+* The `ansible-runner` package is installed
+
+.Procedure
+. Log in to {PlatformName}.
+. In the navigation panel, select *Administration -> Instances*.
+. Click btn:[Add].
+. Input the VM name in the *Host Name* field.
+. *Optional:* Input the port number in the *Listner Port* field.
+. Click btn:[Save].
+. Click the download icon next to *Install Bundle*. This will start a download, take note of where you save the file
```

Comment:
Do you need to add an image of the download icon? There's the Patternfly icon in the images directory (download.png).